server/data-import/data.js

```diff
@@ -31,7 +31,7 @@ async function reimport() {
 	await PregnancyCenterModel.collection.drop()
 
 	try {
-		const dataToImport = fs.readFileSync('server/data-import/test/fixtures/cessilye_nypc.json', 'utf8')
+		const dataToImport = fs.readFileSync('server/data-import/fixtures/cessilye_nypc.json', 'utf8')
```

Comment:
Hi @bhavyaab thank you for making this change! Can you actually put this in a separate PR? That way we can document why we made this specific change. It's good to keep similar changes in the same PR, rather than group a bunch of changes that aren't related. Let me know if you have any questions on this and we can discuss!